packages/core/src/prompts/lesson-assistant/parts/interactive.ts

```diff
@@ -53,13 +59,9 @@ Remove one item from an array:
 Replace a value
 { "op": "replace", "path": "/misconceptions/0/misconception", "value": "A renamed misconception" }
 
-FORMATTING
+The schema of the overall lesson plan is provided below. The changes you make should match the appropriate schema definition for the section you are adding or replacing.
```

Comment:
It says "below". I assume that `lessonPlanJsonSchema` is appended below this prompt part?